packages/cli/src/help/index.ts

```diff
@@ -0,0 +1,42 @@
+import { Help } from '@oclif/core'
+import { BaseCommand } from '@preevy/cli-common'
+
+export default class MyHelpClass extends Help {
+  protected async showRootHelp(): Promise<void> {
+    await super.showRootHelp()
+    this.log(this.formatFlag())
+  }
+
+  protected formatFlag():string {
+    type Flag = {
+      name: string
+      description: string
+    };
+
+    const flags: Flag[] = []
+
+    for (const flagName of Object.keys(BaseCommand.baseFlags)) {
+      if (BaseCommand.baseFlags.hasOwnProperty.call(BaseCommand.baseFlags, flagName)) {
+        const flagKey = flagName as keyof typeof BaseCommand.baseFlags
+        const flag = BaseCommand.baseFlags[flagKey]
+        if (flag.helpGroup === 'GLOBAL') {
+          const flagHelp: Flag = {
+            name: flagName,
+            description: flag.summary ?? '',
+          }
+          flags.push(flagHelp)
+        }
+      }
+    }
+    const body = this.renderList(flags.map(c => [
+      `--${c.name}`,
```

Comment:
There is probably rendering logic implemented in oclif that needs to be reused here.

For example,
- flags with short syntax (--debug, -D)
- boolean flags with the "no-" prefix

Reply:
Hey @royra, So the rendering is in another class in oclif and it is a protected function. The process of creating an instance of that class demands certain constructor parameters that hold no relevance for our current use case. In this case would it be okay if I copy pasted the logic of formatting instead of inheriting? Is there any better way to do it?

Reply:
Which protected function are you referring to?

Reply:
So there is another class in oclif called CommandHelp, which is basically the logic for command help. Oclif shows global flags when --help flag is added to a command and the logic of formatting the flags is in that class itself. 

We essentially need to use the protected flags function:

      protected flags(flags: Array<Command.Flag.Any>)

This function returns the body string in the format we need which we can then render as we are doing already

Reply:
You could inherit from `CommandHelp` and expose it.